.github/workflows/tests.yml

```
@@ -28,4 +28,17 @@ jobs:
       run: go vet -v ./...
 
     - name: Test
-      run: go test -v ./...
\ No newline at end of file
+      run: go test -v ./...
+
+  adapters:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.22'
+
+      - name: Vet
+        run: list -f './adapters' -m | xargs go vet
```

Looks like the command failed, but didn't report a failure 🤔 
```shell
Run list -f './adapters' -m | xargs go vet
  list -f './adapters' -m | xargs go vet
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/6cebb64d-9[2](https://github.com/luno/workflow/actions/runs/10991392493/job/30513649632#step:4:2)db-456d-a572-3f15c68a7e2d.sh: line 1: list: command not found
```
Tried this locally, looks like it'll work (thought `-v` is useful, so you can check it's working too)
```suggestion
        run: go list -f ./adapters/... | xargs go vet -v
```

---

What does `-f` do too? 
Do we need it? Couldn't tell from the help:
```shell
The -f flag specifies an alternate format for the list, using the
syntax of package template. The default output is equivalent
to -f '{{.ImportPath}}'.
```